CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,7 @@
 1. Keep it fun. We're doing serious, at times painstaking, work to make the site incredible, but we can have fun in the process. Summon your best self. Bring a smile.
 2. Differences of opinion are bound to arise. Always give the benefit of the doubt and proceed with kindness. (To put it another way, “Take the high road.”)
 3. Maintain professionalism with one another, the wider KGLW fan community, and members of the band’s organization. Express agreement from team leaders is required when utilizing the kglw.net namesake in community or band outreach.
```

Comment:
I'm inclined to change this one a bit:
* change "KGLW fan community" to "King Gizzard fan community"
* Replace the second sentence with: "It is required to receive explicit agreement from team leaders before utilizing the KGLW.net namesake in community or band outreach."